### PR TITLE
VIH-7753 Change (judge) button shouldn't be displayed during last minute booking

### PR DESCRIPTION
**Pull request by mulvaneyjHMCTS**
### JIRA link (if applicable) ###
VIH-7753

### Change description ###
Removing change button for a judge when editing a hearing inside 30 minutes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.spec.ts [hunk 1]
```diff
@@ -13,8 +14,9 @@ const router = {
 
 const loggerSpy = jasmine.createSpyObj<Logger>('Logger', ['error', 'debug', 'warn']);
 let bookingServiceSpy: jasmine.SpyObj<BookingService>;
+let videoHearingsServiceSpy: jasmine.SpyObj<VideoHearingsService>;
 
-describe('ParticipantItemComponent', () => {
+fdescribe('ParticipantItemComponent', () => {
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.spec.ts:19]: remove f from this describe else only this suite will run

**Review by shaed-parkar — COMMENTED** [review]
don't forget to remove code from testing :)

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.spec.ts (hunk 1 above)
Comment (mulvaneyjHMCTS) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.spec.ts]: Resolved

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.ts [hunk 2]
```diff
@@ -87,4 +93,14 @@ export class ParticipantItemComponent {
...
         return this.participant.is_interpretee;
     }
+
+    canEditJudge(): boolean {
+        if (!this.canEdit || this.videoHearingsService.isConferenceClosed()) {
+            return false;
+        }
+        if (this.videoHearingsService.isHearingAboutToStart()) {
+            return false;
+        }
```
Comment (FarronF) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.ts:103]: These could be combined.

Comment (mulvaneyjHMCTS) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.ts]: Implemented

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.ts [hunk 3]
```diff
@@ -95,10 +95,7 @@ export class ParticipantItemComponent {
     }
 
     canEditJudge(): boolean {
-        if (!this.canEdit || this.videoHearingsService.isConferenceClosed()) {
-            return false;
-        }
-        if (this.videoHearingsService.isHearingAboutToStart()) {
+        if ((!this.canEdit || this.videoHearingsService.isConferenceClosed()) && this.videoHearingsService.isHearingAboutToStart()) {
```
Comment (FarronF) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.ts:98]: Should this not be: 
`!this.canEdit || this.videoHearingsService.isConferenceClosed() || this.videoHearingsService.isHearingAboutToStart()`

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.spec.ts [hunk 4]
```diff
@@ -16,7 +16,7 @@ const loggerSpy = jasmine.createSpyObj<Logger>('Logger', ['error', 'debug', 'war
 let bookingServiceSpy: jasmine.SpyObj<BookingService>;
 let videoHearingsServiceSpy: jasmine.SpyObj<VideoHearingsService>;
 
-describe('ParticipantItemComponent', () => {
+fdescribe('ParticipantItemComponent', () => {
```
Comment (FarronF) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.spec.ts:19]: Remove fdescribe

**Review by FarronF — APPROVED** [review]

**Pull request merged**